lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +207,39 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self.config = config
+        self.urls = []
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        links = article_bs.find_all('a')
+        for link in links:
+            url = link['href']
+            if url.count('/') == 4 and url[0] == '/':
+                url = 'https://www.zebra-tv.ru' + url
+                return url
 
     def find_articles(self) -> None:
         """
         Finds articles
         """
-        pass
+        for url in self.config.get_seed_urls():
+            try:
+                response = make_request(url, config=self.config)
+                soup = BeautifulSoup(response.text, 'lxml')
+                new_url = self._extract_url(soup)
+                if new_url not in self.urls:
+                    self.urls.append(new_url)
+            except HTTPError:
+                continue
 
     def get_search_urls(self) -> list:
         """
         Returns seed_urls param
         """
-        pass
+        return self.config.get_seed_urls()
```

Comment:
it is not the same: seed urls is where you extract article URLs, search urls -> links to articles

Reply:
so i guess the docstring 'returns seed_urls param' should be changed to 'returns urls param'?

Reply:
ok